package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-contracts",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
Should the version bump be its own commit?

Reply:
I also notice that `0.1.4` tag was pushed and auto-deployed to NPM. I think we should include a check deploy will fail if the tag is not on `master`.

Reply:
How would I test the deployment scripts for 
https://github.com/gnosis/dex-contracts-integration-example

Should I do a manual 0.1.4-dev npm release?